pytest/tests/sanity/state_sync_routed.py

```diff
@@ -125,11 +125,15 @@
                                                           boot_heights)
 
 while True:
-    assert time.time(
-    ) - started < TIMEOUT, "Waiting for node 4 to connect to two peers"
-    if metrics4.get_int_metric_value("near_peer_connections_total") == 2:
+    assert time.time() - started < TIMEOUT, "Waiting for node 4 to connect to two peers"
```

Comment:
I noticed that when an assert of this kind fails we don't get to see the assertion message in nayduck output, not sure why...